lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'send_sms'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.